### PR TITLE
Add MK Yuan

### PR DESCRIPTION
**Pull request by fengmk2**
Thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -88,6 +88,7 @@
 - [Vadim Demedes](https://github.com/vdemedes/ama) - Node.js addict.
 - [Jan-Hendrik Heuing](https://github.com/jan-hendrik/ama) - Webdeveloper, Scrum Professional, Agile Coach. Blogging about [Agile and Scrum](https://heuing.io).
 - [Raymond Clanan](https://github.com/rclanan/ama) - Freelance full stack developer at [Utopian Concept LLC](http://utopianconcept.com/).
+- [MK Yuan](https://github.com/fengmk2/ama) - Node.js developer at [Alipay](https://www.alipay.com). Can ask in English or Chinese.
```
Comment (arthurvr) [inline readme.md]: > Can ask in English or Chinese.

Is this something for in the bio? Seems more like something for the AMA's readme instead. WDYT?

**Comment by fengmk2** [comment]
@arthurvr fixed now.

**Comment by arthurvr** [comment]
Welcome!

**Comment by fengmk2** [comment]
Thanks!

**Comment by sindresorhus** [comment]
Welcome! :)

Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=

**Pull request merged**